src/tests/banners/bannerSelection.tsx

```diff
@@ -75,29 +143,29 @@ export const selectBannerTest = (
     targeting: BannerTargeting,
     pageTracking: BannerPageTracking,
     baseUrl: string,
-): Promise<BannerTestSelection | null> => {
-    return redeployedSinceLastClosed(targeting).then(hasRedeployed => {
-        if (hasRedeployed) {
-            const tests = [AusMomentContributionsBanner];
-            const testToRun = tests.find(
-                test =>
-                    targeting.alreadyVisitedCount >= test.minPageViews &&
-                    !targeting.shouldHideReaderRevenue &&
-                    !targeting.isPaidContent &&
-                    test.canRun(targeting, pageTracking),
-            );
-
-            if (testToRun) {
-                const variant = testToRun.variants[0]; // TODO - use mvt
+): BannerTestSelection | null => {
+    const tests = [AusMomentContributionsBanner, SubscriptionsBanner, WeeklyBanner];
+    tests.find(test => {
+        redeployedSinceLastClosed(targeting, test.bannerType).then(hasRedeployed => {
+            if (
+                hasRedeployed &&
+                targeting.alreadyVisitedCount >= test.minPageViews &&
+                !targeting.shouldHideReaderRevenue &&
+                !targeting.isPaidContent &&
+                test.canRun(targeting, pageTracking)
+            ) {
+                const variant = test.variants[0]; // TODO - use mvt
                 return {
-                    test: testToRun,
+                    test,
                     variant,
-                    moduleUrl: `${baseUrl}/${variant.modulePath}`,
-                    moduleName: variant.moduleName,
+                    moduleUrl: variant
```

Comment:
So this is to support tests without any variants? Seems like a good approach, I was wondering whether it might be a bit confusing having `path` and `modulePath` but happy to go with it.

Reply:
👍